ui/lib/pki/addon/components/page/pki-role-details.hbs

```diff
@@ -1,6 +1,6 @@
 <Toolbar>
   <ToolbarActions>
-    {{#if @role.canDelete}}
+    {{#if (or @role.updatePath.isLoading @role.canDelete)}}
```

Comment:
There are other areas of the app where we use this or logic, but I do remember us checking explicitly for false at some point. The way I have it, if the capabilities aren't loaded yet the item will show, and once the value is loaded we can show the value. 

_however_ now that the capabilities values are getters instead of aliases, I think we can put the logic within the getter and that way we can rely on just a single check in the template! I'll update and you can let me know what you think